packages/s/shotcut/package.yml

```diff
@@ -26,11 +26,18 @@ rundeps    :
     - qt5-quickcontrols
```

Comment:
I know this is merged and everything now, but qt5-quickcontrols should be dropped as a rundep. That's from before this package was switched to Qt6.